README.md

```diff
@@ -9,31 +9,42 @@ JWST Calibration Pipeline
 
 Note
 ----
-Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
+Beginning with version 0.9.0, **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest released version of the library, we recommend using [conda](https://conda.io/docs/index.html)
```

Comment:
Perhaps not call it "latest released version" and instead call it "previous night's build" or something similar.  Releases can actually be pulled from the normal `astroconda` channel and don't need this dev stuff.

Reply:
This is the latest 'drop' for the purposes of supplying a release to DMS, so to keep it simple, I'll leave it as is.